lm_eval/__main__.py

```diff
@@ -103,21 +114,23 @@ def parse_eval_args() -> argparse.Namespace:
         "--include_path",
         type=str,
         default=None,
+        metavar="DIR",
         help="Additional path to include if there are external tasks to include.",
     )
     parser.add_argument(
         "--gen_kwargs",
-        default="",
+        default=None,
```

Comment:
Could you check if any part of the code still assumes this as " "?

Reply:
IMO shouldn't be an issue as it's set to `None` anyway if `""` in `simple_evaluate`:
https://github.com/EleutherAI/lm-evaluation-harness/blob/8f5b22955266ed7288b531b8def7509060685114/lm_eval/evaluator.py#L91